safe_core/src/client/mock/mod.rs

```diff
@@ -56,11 +68,17 @@ impl DataId {
         DataId::Mutable(MutableDataId(name, tag))
     }
 
+    /// Create `DataId` for mutable data.
+    pub fn appendonly(name: XorName, tag: u64) -> Self {
```

Comment:
This should be 
```suggestion
    pub fn append_only(name: XorName, tag: u64) -> Self {
```
to be consistent with the other names.